test/cookbook-test.js

```diff
@@ -6,22 +6,28 @@ import Cookbook from '../src/cookbook';
 
 let cookbook;
 
-describe('User', () => {
+describe('Cookbook', () => {
   beforeEach(() => {
     cookbook = new Cookbook(recipeData);
   });
 
+  it('Should be a function', () => {
+   expect(Cookbook).to.be.a('function');
+  });
+
+  it('Should be an instance of Cookbook', () => {
+   expect(cookbook).to.be.an.instanceof(Cookbook)
+  });
+
   it('Should have an array of all recipes', () => {
     expect(cookbook.recipes).to.be.an('array');
   });
 
-  describe('findRecipe', () => {
-    it('Should be able to filter through its array by ingredients', () => {
-      expect(cookbook.findRecipe('yolk').length).to.equal(2);
-    });
+  it('Should be able to filter through its array by ingredients', () => {
+    expect(cookbook.findRecipe('yolk').length).to.equal(2);
+  });
 
-    it('Should be able to filter through its array by name', () => {
-      expect(cookbook.findRecipe('Sesame Cookies').length).to.equal(1);
-    });
+  it('Should be able to filter through its array by name', () => {
```

Comment:
Thanks for adding the missing testing for the Cookbook class, I think we can 2 or 3 sad paths for this specific class